test/test-locales.js

```diff
@@ -0,0 +1,28 @@
+var fs = require('fs');
+var path = require('path');
+var assert = require('assert');
+var glob = require('glob');
+var axe = require(path.join(__dirname, '../axe'));
+
+var localeFiles = glob.sync(path.join(__dirname, '../locales/*.json'));
+
+describe('locales', function() {
+	localeFiles.forEach(function(localeFile) {
+		var localeName = path.basename(localeFile);
+		it(localeName + ' should be valid', function(done) {
+			fs.readFile(localeFile, 'utf-8', function(err, localeData) {
```

Comment:
Why not use the synchronous version? It'd make this a lot easier to read.

Reply:
No particular reason other than I'm use to writing node using async methods

Reply:
Please refactor this to use synchronous `fs` methods then. It'll hugely help readability.